core/controllers/blog_dashboard.py

```diff
@@ -76,13 +76,26 @@ class BlogDashboardDataHandler(base.BaseHandler):
     HANDLER_ARGS_SCHEMAS = {
         'GET': {},
         'POST': {},
+        'PUT': {
+            'displayed_author_name': {
+                'schema': {
+                    'type': 'basestring',
+                }
+            },
+            'author_bio': {
+                'schema': {
+                    'type': 'basestring',
+                }
+            },
+        },
     }
 
     @acl_decorators.can_access_blog_dashboard
     def get(self) -> None:
         """Handles GET requests."""
         user_settings = user_services.get_user_settings(self.user_id)
-
+        author_details = (
+            blog_services.get_blog_author_details(self.user_id).to_dict())
```

Comment:
I'm a bit confused here because in `blog_services.py` you defined `get_blog_author_details` that returns Optional[value] but here you are treating it like it is always going to return a non-Optional value.

If `blog_services.get_blog_author_details(self.user_id)` is None then `blog_services.get_blog_author_details(self.user_id).to_dict()` will raise a python missing attribute error. 

Following the pattern in the codebase, I think you should define an ` if blog_services.get_blog_author_details(self.user_id) is None: raise Exception` clause. Please let me know if you need any help or more info.

Reply:
Function will not return None, removed Optional[..] from return of `get_blog_author_details()` and added a raise exception condition in `get_blog_author_details()`.